docs/generation.rst

```diff
@@ -3,122 +3,91 @@
+      --unstable \
+      https://deno.land/x/edgedb/generate.ts <generator> [options]
 
-.. code-block:: bash
+The value of generators should be one of the following:
 
-  $ deno run --allow-all --unstable https://deno.land/x/edgedb/generate.ts <generator> [options]
+.. list-table::
+   :class: funcoptable
 
+   * - ``edgeql-js``
+     - Provides a **code-first** way to write **fully-typed** EdgeQL queries
+       with TypeScript. We recommend it for TypeScript users, or anyone who
+       prefers writing queries with code.
+     - :ref:`docs <edgedb-js-qb>`
 
-The value of ``<generator>`` should be one of the following.
+   * - ``queries``
+     - Scans your project for ``*.edgeql`` files and generates functions that
+       allow you to execute these queries in a typesafe way.
+     - :ref:`docs <edgedb-js-queries>`
 
-- ``queries``: This generator scans your project for ``*.edgeql`` files and generates a file containing a strongly-typed function alongside each. Alternatively you can use ``--file`` mode to generate a single file containing all the "query functions".
-- ``edgeql-js``: This generator introspects your database schema and generates a query builder.
-- ``interfaces``: This generator introspects your database schema and generates TypeScript interfaces for each object type.
+   * - ``interfaces``
+     - Introspects your schema and generates file containing *TypeScript
+       interfaces* that correspond to each object type. This is useful for
+       writing typesafe code to interact with EdgeDB.
+     - :ref:`docs <edgedb-js-interfaces>`
 
 Connection
 ^^^^^^^^^^
 
-Generating the query builder requires a
-connection to an active EdgeDB database. It does **not** simply read your local ``.esdl`` schema files. Generators rely on the database to introspect the schema and analyze queries. Doing so without a database connection would require implementing a full EdgeQL parser and static analyzer in JavaScript—which we don't intend to do anytime soon.
+Generating the query builder requires a connection to an active EdgeDB database.
+It does **not** simply read your local ``.esdl`` schema files. Generators rely
+on the database to introspect the schema and analyze queries. Doing so without a
+database connection would require implementing a full EdgeQL parser and static
+analyzer in JavaScript—which we don't intend to do anytime soon.
 
 .. note::
 
-  Make sure your development database is up-to-date with your latest schema before running a generator!
+  Make sure your development database is up-to-date with your latest schema
+  before running a generator!
 
-If you're using ``edgedb project init``, this is automatically handled for you. Otherwise, you'll need to explicitly pass connection information via environment variables or CLI flags, just like any other CLI command. See :ref:`Client Libraries > Connection <edgedb_client_connection>` for guidance.
+If you're using ``edgedb project init``, this is automatically handled for you.
```

Comment:
Be explicit about what "this" refers to here. I think you're referring to "connecting to your instance"?